products/jbrowse-cli/README.md

```diff
@@ -471,6 +471,9 @@ OPTIONS
 
   --perTrack                   If set, creates an index per track
 
+  --prefixSize=prefixSize      [default: 6] Specify the prefix size for the ixx index, increase size if many of your
+                               gene IDs have same prefix e.g. Z000000001, Z000000002
```

Comment:
@cmdcolin I just noticed this defaults to 6 but ixIxx defaults to 5. We should probably default to the same one.

Reply:
that is true. it will not cause any issue the way it is now since ixixx doesn't use it's default value, it gets overridden by 6 here, but I can change ixixx to be 6 by default